skbio/embedding/_protein.py

```diff
@@ -0,0 +1,97 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, scikit-bio development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+from skbio.sequence import Protein
+from skbio.embedding._embedding import SequenceEmbedding
+from skbio.util import get_data_path
+from pathlib import Path
+import pandas as pd
+import numpy as np
+
+
+class ProteinEmbedding(SequenceEmbedding):
```

Comment:
May also be nice to have a simple boolean like `clip_head` but to check if the input sequence has characters separated by spaces, and if it does, do `sequence_example = sequence_example.replace(" ", "")` or something similar. Not a huge deal for the user to do this themselves, but some models like `prot_bert` need the sequences to have spaces in between characters, where this class currently assumes that there are no spaces between characters.

Reply:
I added this in